caom2pipe/execute_composable.py

```diff
@@ -1381,11 +1398,7 @@ def _unset_file_logging(self):
             self._log_h.close()
 
     def can_use_single_visit(self):
-        return len(self.task_types) > 1 and (
-            (mc.TaskType.STORE in self.task_types and mc.TaskType.INGEST in self.task_types)
-            or (mc.TaskType.INGEST in self.task_types and mc.TaskType.MODIFY in self.task_types)
-            or (mc.TaskType.SCRAPE in self.task_types and mc.TaskType.MODIFY in self.task_types)
-        )
+        return can_use_single_visit(self.task_types)
```

Comment:
Method still required? It can get confusing to have a method and a function with the same name.

Reply:
I have use cases for the function, and use cases for the method, and the condition is complex enough that I do not want to duplicate it. Would it be better to rename either the method or the function?